tensorflow/src/main/scala/magnolify/tensorflow/ExampleType.scala

```diff
@@ -119,30 +120,23 @@ object ExampleField {
         f
       }
 
-    override protected def buildSchema(k: String)(cm: CaseMapper): Schema = {
-      val sb = Schema
-        .newBuilder()
+    override protected def buildSchema(cm: CaseMapper): Schema = {
+      val sb = Schema.newBuilder()
       getDoc(caseClass.annotations, caseClass.typeName.full).foreach(sb.setAnnotation)
-
       caseClass.parameters.foldLeft(sb) { (b, p) =>
-        val s = p.typeclass.buildSchema(key(k, cm.map(p.label)))(cm)
-        val annotatedFs = s.getFeatureList.asScala
-          .map { f =>
-            val maybeAnnotation = if (f.hasAnnotation) {
-              Some(f.getAnnotation)
-            } else {
-              getDoc(p.annotations, s"${caseClass.typeName.full}#${key(k, cm.map(p.label))}")
-            }
-
-            maybeAnnotation match {
-              case Some(a) => f.toBuilder.setAnnotation(a).build()
-              case None    => f
-            }
-          }
-        b.addAllFeature(annotatedFs.asJava)
+        val n = cm.map(p.label)
```

Comment:
Appreciate the terseness but maybe longer variable names would be more interpretable?